tests/ja4-rules/test.yaml

```diff
@@ -27,3 +28,6 @@ checks:
       match:
         event_type: alert
         alert.signature: JA4 TLS Test 1
+  - shell:
```

Comment:
Should this new check be in its own commit ?

Reply:
Yes. I'll prepare a new branch/pr